mix/MixClient.cpp

```diff
@@ -40,7 +40,7 @@ namespace mix
 {
 
 Secret const c_defaultUserAccountSecret = Secret("cb73d9408c4720e230387d956eb0f829d8a4dd2c1055f96257167e14e7169074");
-u256 const c_mixGenesisDifficulty = c_minimumDifficulty; //TODO: make it lower for Mix somehow
```

Comment:
Maybe this constant is used in AZ or eth. So the value will be 0 in the windows build ...